openedx/features/cms/helpers.py

```diff
@@ -62,3 +78,66 @@ def set_rerun_course_module_dates(source_course, re_run, user):
             meta_data['due'] = re_run_start_date - due_date_delta
 
         _save_xblock(user, re_run_xblock, metadata=meta_data)
+
+
+def set_rerun_ora_dates(re_run_subsections, re_run_start_date, source_course_start_date, user):
+    """
+    This method is responsible for updating all dates in ORA i.e submission, start, due etc, for
+    the re-run according to source course. This is achieved by calculating new dates for ORA based
+    on delta value.
+    :param re_run_subsections: list of subsection in a (re-run) course
+    :param re_run_start_date: course start date of source course
+    :param source_course_start_date: course start date of source course
+    :param user: user that created this course
+    """
+    def update_date_by_delta(date_to_update, default_date):
```

Comment:
lets change `update_date_by_delta` => `compute_ora_date_by_delta`
since we're not updating we're computing